tensorflow/lite/experimental/micro/micro_interpreter.cc

```diff
@@ -96,6 +96,51 @@ MicroInterpreter::MicroInterpreter(const Model* model,
   if (initialization_status_ != kTfLiteOk) {
     return;
   }
+
+  // If the system is big endian then convert weights from the flatbuffer from
+  // little to big endian on startup so that it does not need to be done during
+  // inference.
```

Comment:
Can you add a comment clarifying that this assumes the weight parameters are held in memory that can be modified, rather than being truly read-only?

Reply:
Added an additional note here, that the memory must be able to be modified.